curp/src/rpc/connect.rs

```diff
@@ -583,9 +636,37 @@ where
         _timeout: Duration,
     ) -> Result<tonic::Response<FetchReadStateResponse>, CurpError> {
         let mut req = tonic::Request::new(request);
+        req.metadata_mut().inject_bypassed();
         req.metadata_mut().inject_current();
         self.server.fetch_read_state(req).await.map_err(Into::into)
     }
+
+    /// Keep send lease keep alive to server and mutate the client id
+    async fn lease_keep_alive(&self, _client_id: Arc<RwLock<u64>>) -> CurpError {
+        unreachable!("cannot invoke lease_keep_alive in bypassed connect")
+    }
+}
+
+/// Generate heartbeat stream
+fn heartbeat_stream(client_id: Arc<RwLock<u64>>) -> impl Stream<Item = LeaseKeepAliveMsg> {
+    /// Keep alive interval
+    const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(1);
```

Comment:
Shall we make this interval configurable?

Reply:
Fixed in #617